### DISCUSSION TITLE
Worse results than before in quality of generated images

### DISCUSSION THREAD

**Discussion by thvaz** [lllyasviel/Fooocus#2827] [General]
I dont know it it was an update, a problem with my setup, but It seems the quality of the generated images are a lot worse than what I would get some weeks ago.The results I have been getting are very bad faces, weird textures, an overall drop of quality in general. Didn' change anything at all.

**Comment by ILikeToasters** [comment]
No changes have been made to Fooocus that would impact that. I would recommend going back to some images you generated in the past and try to recreate them with the same seed and settings to see if you get the same image to verify things. Have you tried that?

**Comment by wisdombox44** [comment]
Try with lightning preset, some times less steps gives better results.
